zerver/tests/test_invite.py

```diff
@@ -1507,7 +1509,10 @@ def test_send_more_than_one_invite_to_same_user(self) -> None:
         lear = get_realm("lear")
         lear_user = self.lear_user("cordelia")
         do_invite_users(
-            lear_user, ["foo@zulip.com"], [], invite_expires_in_minutes=invite_expires_in_minutes
+            lear_user,
+            ["foo@zulip.com"],
+            [],
+            invite_expires_in_minutes=invite_expires_in_minutes,
```

Comment:
Is this formatting change required? The arguments passed to function are same as before.